onadata/settings/kc_environ.py

```diff
@@ -144,6 +144,15 @@
     if not os.path.isdir(EMAIL_FILE_PATH):
         os.mkdir(EMAIL_FILE_PATH)
 
+# Local time zone for this installation. Choices can be found here:
+# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
+# although not all choices may be available on all operating systems.
+# On Unix systems, a value of None will cause Django to use the same
+# timezone as the operating system.
+# If running in a Windows environment this must be set to the same as your
+# system time zone.
+TIME_ZONE = os.environ.get('TIME_ZONE', 'America/New_York')
```

Comment:
> On Unix systems, a value of None will cause Django to use the same timezone as the operating system.

But there is no way to set this to None, is it? In case of an unset variable, this will return 'America/New_York', so how to set it to None?